consensus-types/blocks/proto.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 // Proto converts the signed beacon block to a protobuf object.
-func (b *SignedBeaconBlock) Proto() (proto.Message, error) {
+func (b *SignedBeaconBlock) Proto() (proto.Message, error) { // nolint:gocognit
```

Comment:
probably a todo to improve

Reply:
I dont think we can really make this much less complex :(